### ISSUE TITLE
Misleading doc : actionButtonStyle

### ISSUE THREAD

**Issue by wer-mathurin** [bahricanyesil/flutter-animated-login#8]
This style only affect the login  
/// Custom button style for action button (login/signup).
  final ButtonStyle? actionButtonStyle;

And this one affect the registration button (maybe rename it for more clarity
changeActionButtonStyle

Cheers,

**Comment by bahricanyesil** [comment]
Actually, the situation is like this:

When you are in the login mode, the "login" button will be the "action button" and on the other side (that has a blue background), the "signup" button will be the "change action button". However, if you click "signup" (change action button) in login mode. The screen will switch between these modes with animation. Then, on the side that has a blue background (I called it as "welcome" part), the "change action button" will have the "login" text on it. The button under the form will have "signup".

In short: texts on the buttons are dynamically changing while switching between modes. The types of buttons are determining their styles, colors, and so on in this concept. The action button is the button below the form always and the change action button is the button on the blue background.

So, "actionButtonStyle" doesn't only affect the login, it affects login in login mode. However, if you switch between modes (by clicking the register button that you indicated), now it will affect the "signup".

Thanks,
